packages/database/lib/migrations/20250825230800_reset_getting_started.cjs

```diff
@@ -0,0 +1,37 @@
+exports.config = { transaction: true };
+
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    // Soft-delete all connections referenced in getting_started_progress
+    await knex('_nango_connections')
+        .whereIn('id', function () {
+            this.select('connection_id').from('getting_started_progress').whereNotNull('connection_id');
+        })
+        .update({
+            deleted: true,
+            deleted_at: knex.fn.now()
+        });
```

Comment:
I wish it were that simple, but since connections can have scripts, it need to be correctly deleted. Not sure what's best, but ultimately I wouldn't do it in the DB migration.

Reply:
Yeah I totally understand. In reality, the migration is run in another pod, while the current pod is still on the previous version. So running it in migration is just a way to run as close as possible to the swap, but it's not bulletproof either. 
I would just deploy and run a script locally (alternative: push the script with this PR, and run it in the pod).
For a few seconds some getting started won't work, I don't think it's a massive issue, especially that you can't avoid it anyway